mule/docs/palette-item-mappings-v4.md

```
@@ -4539,8 +4535,62 @@ service /foo on config {
     }
 }
 
-function _dwMethod(Context ctx) returns json {
-    return "apple".toUpperAscii();
+public function _dwMethod(Context ctx) returns json => "apple".toUpperAscii();
+
+```
+
+- ### Transform Message With Dw Parsing Failure
```

markdownlint-cli2 (0.21.0)

[warning] Heading style
Expected: atx_closed; Actual: atx

(MD003, heading-style)